node/polka/app.js

```diff
@@ -5,6 +5,9 @@ polka()
     res.end('')
   })
   .get('/user/:id', (req, res) => {
+    if (!(/^\d+$/.test(req.params.id))) {
+      req.abort()
```

Comment:
This is NOT what I PR'd and is both an invalid response handler & is shit performance. https://github.com/the-benchmarker/web-frameworks/pull/1529/files

Thanks -.-

Reply:
yep, that's a diry fix I made. yours is still not pushed :heart: